### ISSUE TITLE
Misplaced axes label for vcs plots

### ISSUE THREAD

**Issue by chengzhuzhang** [E3SM-Project/e3sm_diags#219]
Below are two examples that unexpected labels shown up for x and y axes

![MERRA2-T-ANN-global](https://user-images.githubusercontent.com/13056557/58197871-00a64480-7c82-11e9-8edc-9ff9333beeb8.png)
![GPCP_v2 2-PRECT-ANN-global](https://user-images.githubusercontent.com/13056557/58197872-013edb00-7c82-11e9-85f1-7c0c20207f3b.png)

**Comment by doutriaux1** [comment]
@zshaheen @chengzhuzhang  you will need to turn the labels off in your template.

**Comment by zshaheen** [comment]
@doutriaux1, we haven't changed the vcs code in some time. FYI, this wasn't an issue with older versions of vcs. We are using the following versions of vcs and its related software:
```
  - vcs=8.1=py_0
  - vcsaddons=8.0=py36_0
  - vtk-cdat=8.2.0rc2.8.1=py36h413fb2d_0
```
This is also **not** something caused due to any Python 2 vs Python 3 differences. According to @chengzhuzhang, both Python 2 and Python 3 versions of this vcs are causing this issue.

**Comment by doutriaux1** [comment]
@zshaheen it is actually due to a `bug fix` in templates these were never rendered before, so now that the bug is fixed the units appear on your plot 😄

**Comment by zshaheen** [comment]
@doutriaux1, great, I will fix this.

**Comment by zshaheen** [comment]
@doutriaux1, how does one programmatically do this? As in, in the isofill objects, what attribute is it whose priority needs to be 0? I can't seem to figure out which one it is in the below.

```
['__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__',
'__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__',
'__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__slots__', '__str__',
'__subclasshook__', '__weakref__', '_colormap', '_datawc_calendar', '_datawc_timeunits', '_datawc_x1',
'_datawc_x2', '_datawc_y1', '_datawc_y2', '_ext_1', '_ext_2', '_fillareacolors', '_fillareaindices', 
'_fillareaopacity', '_fillareapixelscale', '_fillareapixelspacing', '_fillareastyle', '_getcalendar', 
'_getdatawc_x1', '_getdatawc_x2', '_getdatawc_y1', '_getdatawc_y2', '_getfillareacolors',
'_getfillareaindices', '_getfillareastyle', '_getmissing', '_getname', '_gettimeunits', '_getxaxisconvert', 
'_getxmtics1', '_getxmtics2', '_getxticlabels1', '_getxticlabels2', '_getyaxisconvert', '_getymtics1', 
'_getymtics2', '_getyticlabels1', '_getyticlabels2', '_legend', '_levels', '_missing', '_name', '_projection', 
'_setcalendar', '_setdatawc_x1', '_setdatawc_x2', '_setdatawc_y1', '_setdatawc_y2', '_setfillareacolors', 
'_setfillareaindices', '_setfillareastyle', '_setmissing', '_setname', '_settimeunits', '_setxaxisconvert', 
'_setxmtics1', '_setxmtics2', '_setxticlabels1', '_setxticlabels2', '_setyaxisconvert', '_setymtics1', 
'_setymtics2', '_setyticlabels1', '_setyticlabels2', '_xaxisconvert', '_xmtics1', '_xmtics2', '_xticlabels1', 
'_xticlabels2', '_yaxisconvert', '_ymtics1', '_ymtics2', '_yticlabels1', '_yticlabels2', 'colormap', 'colors', 
'datawc', 'datawc_calendar', 'datawc_timeunits', 'datawc_x1', 'datawc_x2', 'datawc_y1', 'datawc_y2', 
'ext_1', 'ext_2', 'exts', 'fillareacolors', 'fillareaindices', 'fillareaopacity', 'fillareapixelscale', 
'fillareapixelspacing', 'fillareastyle', 'g_name', 'info', 'legend', 'levels', 'list', 'missing', 'name', 
'projection', 'script', 'xaxisconvert', 'xmtics', 'xmtics1', 'xmtics2', 'xticlabels', 'xticlabels1', 'xticlabels2', 
'xyscale', 'yaxisconvert', 'ymtics', 'ymtics1', 'ymtics2', 'yticlabels', 'yticlabels1', 'yticlabels2']

```

**Comment by doutriaux1** [comment]
it's on the template see: https://github.com/CDAT/Jupyter-notebooks/blob/vcs_templates/vcs/VCS_Templates/VCS_Templates.ipynb

**Comment by zshaheen** [comment]
This was fixed in 76c0f4eb38df16c9c72f249dbfd87a07574b7812. See the below image. Thanks for the help Charles.

![ERA-Interim-T-850-ANN-global-1](https://user-images.githubusercontent.com/4239938/58207044-c5624080-7c96-11e9-9240-205c8a1069cf.png)
